frontend/src/components/DestructionListAuditLog/DestructionListAuditLogDetails.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { expect, within } from "@storybook/test";
+
+import { auditLogItemFactory } from "../../fixtures/auditLog";
+import { userFactory } from "../../fixtures/user";
+import { DestructionListEditPage } from "../../pages";
+import { DestructionListAuditLogDetails } from "./DestructionListAuditLogDetails";
+
+const meta: Meta<typeof DestructionListEditPage> = {
```

Comment:
This is incorrect in many places, but I'm fixing it. Can you set it to DestructionListAuditLogDetails here?